static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-epic.html',
```

Comment:
I think giraffe-message.html and contributions-epic.html should be combined but if you look at contributions-embed, it is really quite different, for example I had to put a container div around the buttons and the contribution button because their orientation changed depending on which variant of the test it was running, plus it's wrapped in a figure tag, which is not necessary for the other 2. Could I just combine the first 2?